packages/service-library/src/web-api/api-controller.ts

```diff
@@ -74,7 +74,7 @@ export abstract class ApiController extends WebController {
             return false;
         }
 
-        if (this.context.req.query['api-version'] !== this.apiVersion) {
+        if (this.context.req.query['api-version'] !== this.apiVersion && this.context.req.query['api-version'] !== '2.0') {
```

Comment:
We can change this.apiVersion to array since the single controller implementation supports two versions

Reply:
Synced offline - This is okay for now.